guides/source/action_mailer_basics.md

```diff
@@ -819,8 +843,8 @@ end
 [`before_action`]: https://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-before_action
 [`before_deliver`]: https://api.rubyonrails.org/classes/ActionMailer/Callbacks/ClassMethods.html#method-i-before_deliver
 
-Using Action Mailer Helpers
----------------------------
+Action Mailer Helpers
+---------------------
 
 Action Mailer inherits from `AbstractController`, so you have access to most
 of the same helpers as you do in Action Controller.
```

Comment:
This is somewhat confusing to me. I'm guessing the helpers in this case are ActionView helpers?
Also, how the helpers are included isn't relevant I guess (it could change as well).
```suggestion
Action Mailer views have access to most
of the same helpers as regular views.
```